modules/gapi/src/streaming/onevpl/accelerators/accel_policy_dx11.cpp

```diff
@@ -283,12 +293,24 @@ mfxStatus VPLDX11AccelerationPolicy::on_alloc(const mfxFrameAllocRequest *reques
         desc.BindFlags = 0;
     }
 
+    /* NB:
+     * On the one hand current OpenVINO API doesn't support texture array and
+     * D3D11 API doesn't allow to address specific texture element in array.
+     * On the other hand using textures array should be more performant case
+     * in applications (according to community experience)
+     * So, to be compliant with OV let's turn off textures array feature, but keep
+     * this code in commented section to consider such "optimization" in future
+     */
+    /*
```

Comment:
i doubt if it would be activated in the near future... it arises as integration obstacles between VPL & OV.
But because OV is a most desirable target of such integration this is done for OV purposes

Maybe you right and MACRO would be more preferrable here